mautic/mautic
# Text message type visible in categories page

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [ Yes]
| New feature/enhancement? (use the a.x branch)      | [ NA]
| Deprecations?                          | [ NA]
| BC breaks? (use the c.x branch)        | [No ]
| Automated tests included?              | [ No] 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #... 



#### Description:


Category type was not showing up for SMS on categories page. This PR fixes that. 
<img width="589" alt="image" src="https://github.com/mautic/mautic/assets/45942361/fff4e477-c33b-4a45-b920-80aed56f9756">


#### Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Configure Twilio plugin to enable Text Messages.
3. Go to Text Messages and click on New button to create text message.
4. Select "New Template Text Message" type 
<img width="582" alt="image" src="https://github.com/mautic/mautic/assets/45942361/f783d8f7-930f-4736-af9e-5adc63e6523c">

5. Select create category from category dropdown.
6. Create new category
7. Fill required fields for text message and save it.
8. Go to category list page you will see the populated type column for newly created category 
<img width="1446" alt="image" src="https://github.com/mautic/mautic/assets/45942361/ae634195-34f9-49de-b15c-800750e40413">
